normalView/normalView.js

```diff
@@ -0,0 +1,34 @@
+let button = document.getElementById("button1");
+let notification="on";
+
+localStorage.getItem("notification",notification);
+
+
+
+//localStorage.setItem("count",count);
+button.onclick= function() { 
+    // var property = document.getElementById("button");
+     // if paused start
+
+
+      if (notification === "on") {
+              notification="off";   
+              button.innerHTML = "Start";
+   
```

Comment:
Indentation is very important, not only keeps the code tidy but it makes easier to spot where the functions start and end, and read the code.